src/patternfly/layouts/Page/docs/code.md

```diff
@@ -7,6 +7,7 @@ This layout provides the basic chrome for a page, including sidebar, header and
 | Attribute | Applied To | Outcome |
 | -- | -- | -- |
 | `role="banner"` | `.pf-l-page__header` | .  |
+| `role="sidebar"` | `.pf-l-page__sidebar` | .  |
```

Comment:
you're acutally using `role="aside"` but should this be `role="complementary"` instead?

Reply:
Good catch, @michael-coker! Here's a link to the implicit roles for html elements: https://www.w3.org/TR/html-aria/

`<aside>` has the implicit role of `complementary`. So really we could just remove the explicit role definition altogether.

Reply:
Updated